codes/auto_test.py

```diff
@@ -1,32 +0,0 @@
-'''auto test several models.'''
```

Comment:
Undo delete, while the file has not been modified in a while, it's used to test multiple models one after the other, can be updated

Reply:
Done.